Cargo.lock

```diff
@@ -3822,6 +3850,16 @@ version = "0.2.8"
 source = "registry+https://github.com/rust-lang/crates.io-index"
 checksum = "4ec2a862134d2a7d32d7983ddcdd1c4923530833c9f2ea1a44fc5fa473989058"
 
+[[package]]
+name = "libredox"
```

Comment:
Would it be possible to remove `libredox` and `redox_users` deps? I don't think we want to pull these in

Reply:
That comes in via https://crates.io/crates/dirs. I could probably re-implement what's needed directly, unless you have other ideas.

Reply:
ah, its optional, only compiled on redox target https://github.com/dirs-dev/dirs-sys-rs/blob/c0fd66cb08f1f97ebf670914253a34bd42d284fb/Cargo.toml#L17-L18. Should be fine